sjb/actions/post_action.py

```diff
@@ -12,4 +12,4 @@ def __init__(self, action):
         self.action = action
 
     def generate_post_build_steps(self):
-        return self.action.generate_build_steps()
+        return self.action.generate_build_steps("xml")
```

Comment:
Why hard coded?

Reply:
Skipped this. Post actions are skipped in generated shell scripts